pkg/repo/migrations/helpers.go

```diff
@@ -141,3 +143,18 @@ func copyFile(srcPath, dstPath string) error {
 
 	return nil
 }
+
+// writeInstallationID writes the installation ID to system wide config path
+func writeInstallationID(cfg system.GlobalConfig, installationID string) error {
+	// Create config dir if it doesn't exist
+	if err := os.MkdirAll(cfg.ConfigDir(), util.OS_USER_RW); err != nil {
+		return fmt.Errorf("creating config dir: %w", err)
+	}
+
+	// Write installation ID to file
+	installationIDFile := filepath.Join(cfg.ConfigDir(), system.InstallationIDFile)
+	if err := os.WriteFile(installationIDFile, []byte(installationID), util.OS_USER_RW); err != nil {
+		return fmt.Errorf("writing installation ID file: %w", err)
+	}
+	return nil
+}
```

Comment:
nit: I would prefer we accept the path as a parameter rather than the `GlobalConfig` interface.